README.md

```diff
@@ -1,14 +1,24 @@
-![Alt text](logos/orbit_logo_simple.png)
+# ORBIT
 
-![Alt text](logos/screenshot.png)
+<img alt="ORBIT Logo" src="logos/orbit_logo_simple.png" align="right" width="520" >
 
-Orbit, the **O**pen **R**untime **B**inary **I**nstrumentation **T**ool, is a standalone **C/C++** profiler for Windows and Linux. Its main purpose is to help developers visualize the execution flow of a complex application.
+<!--
+Don't let the text wrap too narrowly to the left of the above image.
+The `div` reduces the vertical height.
+GitHub will autolink `img`, but won't produce a link when `href="#"`.
+-->
+<div><a href="#"><img src="contrib/docs/images/bumper.png" width="250"></a></div>
```

Comment:
We should use the `======` or `------` markdown instruction instead of the bumper.
In particular, what are the licence for that bumper.png? Where is this coming from?

Reply:
Yeah. The `bumper.png` is licensed (open-source) to Google, had taken it from carbon-lang repo.
Will delete it, let's keep things clutter free.